src/cpp/shared/adventure/hero.cpp

```diff
@@ -14,7 +14,7 @@ char cHeroTypeInitial[13] ={'k', 'b', 's', 'w', 'z', 'n',
 	                        '\0','\0','\0','\0','\0','\0',
                             'c'};
 
-signed char gHeroSkillBonus[NUM_FACTIONS][2][4] = {
+signed __int8 gHeroSkillBonus[NUM_FACTIONS][2][4] = {
```

Comment:
__int8 x = '#';

Can you tell me what's wrong with this?

You can change this array to __int8 in IDA; then the decompiler should generate this array for you with numbers instead of characters.

Reply:
It didn't occur to me that I should change the data to integer values instead of char values. I think this works anyways, but I'll go ahead and change the data to integer values instead, since it would look better.

Reply:
It was already defined as an __int8 in IDA, so I'm not sure why it decompiled as characters. I'm taking the values from IDA and entering them by hand.